src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React, { useEffect, useState } from 'react';
+import { getPeople } from '../../api/people';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+interface Error {
+  status: boolean,
+  message: string,
+}
```

Comment:
You don't need error to be an object. Simple string with message will fit all your needs.